script.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+sudo dnf update
+sudo dnf install https://repo.mysql.com//mysql80-community-release-fc30-1.noarch.rpm
+sudo dnf install mysql-community-server
+
+systemctl start mysqld
```

Comment:
How do you verify the service is persistent between restarts?

Reply:
When you do changes following a review comment, do add your feedback in the conversation.

Reply:
why did you remove line 8?

Reply:
got it, working on it

Reply:
worked on the file, submitted chenges

Reply:
I don't see where you addressed the persistance issue. Have you tested it?